test/tests/table.ts

```diff
@@ -117,13 +146,14 @@ suite('Table', () => {
     })
 
     suite('query', () => {
-        test('should allow you to chain index query statements', async () => {
-            const tableCursor = decentiumTrendingTable.query({from: 5, to: 10}).query({to: 8})
-            assert.deepEqual(
-                Serializer.objectify(await tableCursor.all()).map((row) => row.id),
-                [5, 6, 7, 8]
-            )
-        })
+        // TODO: Discuss and remove? No longer works since query is a Table method
+        // test('should allow you to chain index query statements', async () => {
```

Comment:
Yeah, I guess we can just remove this. I personally kind of liked the fact that you could chain query statements like this, but given that the cursor and table classes are now built to be used independently, it no longer really makes sense to have a query method on the cursor.

Reply:
Removed for now, and left a link to this discussion in the commit.